### PR TITLE
Create serverless-api-gateway.yml

### PR DESCRIPTION
**Pull request by irensaltali**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

**Comment by kokomo123** [comment]
Feel free to get rid of all the comments as they aren't needed 👍🏻

**Review by kokomo123 — APPROVED** [review]
LGTM, No issues here pending other thoughts from collaborators. Thanks for your contribution!

File: licenses.yml [hunk 1]
```diff
@@ -71,6 +71,10 @@
   name: Common Public Attribution License Version 1.0
   url: https://spdx.org/licenses/CPAL-1.0.html
 
+- identifier: Elastic-2.0
+  name: Elastic License 2.0
+  url: https://spdx.org/licenses/Elastic-2.0.html
+
```
Comment (nodiscc) [inline licenses.yml:77]: Elastic License is not Free and Open-Source https://spdx.org/licenses/

~~This block should be moved to `licenses-nonfree.yml`~~ Actually it's already there so no need to add it again

**Review by nodiscc — CHANGES_REQUESTED** [review]

File: licenses.yml (hunk 1 above)
Comment (irensaltali) [reply licenses.yml]: I removed it.

**Comment by Rabenherz112** [comment]
Does this qualify as self-hosted, if deploy / hosted the worker on Cloudflare instead of your own hardware?

**Comment by nodiscc** [comment]
> Does this qualify as self-hosted

I don't think it does.

**Pull request closed without merging**